docs/design/initial-design.md

```diff
@@ -60,14 +60,12 @@ operation in the Data Mover controller. The main things that this must
 contain are a list of PVCs to back up and a `StorageLocation`
 reference.
 
-The main open question here is whether an explicit list of PVCs be
-the only way to identify what to back up.  Will we also allow a list
-of namespaces (including all PVCs in each namespace)? What about
-allowing the use of label selectors? If namespace or label selectors
-are used, we'll also need a Status field to provide the full PV list.
-If these alternate means are supported, do we support all 3 at once
-and take the union of the matches, or do we require that if one is
-specified the others are left blank?
+The main open question here is whether an explicit list of PVCs be the
+only way to identify what to back up.  Will we also allow a list of
+namespaces (including all PVCs in each namespace)? What about allowing
+the use of label selectors? If these alternate means are supported, do
```

Comment:
I think supporting label selectors make sense here. A list of PVCs will be annoying to present to data mover since I need to include a name+ns reference. I optionally like specifying a list of namespaces here as well and then we could take the union of the two options.

Reply:
Actually there are 3 options. Explicit PVC list, "include all in namespaces" list, and label selector. And yes, it probably makes sense to take the union of all 3.